test/deno/web.test.ts

```diff
@@ -1,17 +1,17 @@
-import { sign, verify } from "../../src/web.ts";
+import { sign, verify } from "../../pkg/dist-web/index.js";
 
 import { assertEquals } from "std/testing/asserts.ts";
 
 Deno.test("sign", async () => {
   const actual = await sign("secret", "data");
   const expected =
-    "1b2c16b75bd2a870c114153ccda5bcfca63314bc722fa160d690de133ccbb9db";
+    "sha256=1b2c16b75bd2a870c114153ccda5bcfca63314bc722fa160d690de133ccbb9db";
   assertEquals(actual, expected);
 });
 
 Deno.test("verify", async () => {
   const signature =
-    "1b2c16b75bd2a870c114153ccda5bcfca63314bc722fa160d690de133ccbb9db";
```

Comment:
we shouldn't need to change existing tests, maybe add a new test instead? Current behavior should not change

Reply:
Can you share the reasoning for that? It does break existing functionality, however it brings it inline with the node version

Reply:
sorry, I remember now, ignore my comment.